tests/integration/scripteditor.ts

```diff
@@ -109,6 +109,20 @@ describe('Script Editor tests', () => {
     cy.get('button.jp-mod-warn').click();
   });
 
+  it('opens new output console', () => {
+    openFile('py');
+    cy.get('button[title="Run"]').click({ force: true });
```

Comment:
I think you can remove the `{ force: true }` option since there should be only one element found at this point.